Fixtures/TestProject/TestProject.xcodeproj/project.pbxproj

```diff
@@ -102,9 +104,20 @@
 			path = TestProjectTests;
 			sourceTree = "<group>";
 		};
+		CC9392641F76EDE600C1934A /* Configs */ = {
```

Comment:
How did you create this `TestProject.xcodeproj/project.pbxproj` ?
On Xcode, `Configs` group is missing reference (appears red).

Reply:
The `TestProject.xcodeproj` was created and edited in Xcode. `GeneratedProject.xcodeproj` is the one generated by XcodeGen.

I'm not sure what you mean by missing configs group. Both projects appear fine for me

Reply:
I just saw what you mean by the red group, I've fixed this in master. This project isn't really used in any tests and is more just for a comparison between a project created in xcode versus created in XcodeGen